configuration/src/configuration.ts

```diff
@@ -5,10 +5,12 @@ import {FileConfiguration} from './fileConfiguration';
 import {EnvConfiguration} from './envConfiguration';
 import {MongoConfiguration} from './mongoConfiguration';
 import {DefaultConfiguration} from './defaultConfiguration';
+import {KeyVaultConfiguration, KeyVaultConfigurationOptions} from './keyVaultConfiguration';
 
 const mongoUriKey: string = 'MONGO_URI';
 const mongoDbNameKey: string = 'MONGO_CONFIG_DB';
 const mongoCollectionNameKey: string = 'MONGO_CONFIG_COLLECTION';
+const keyVaultConfigKey = 'KEYVAULT';

```

Comment:
The KeyVault config doesn't contain any secrets, so it can be passed in by other config providers (env, mongo, or user-config.json).